src/SingleProject/Resizetizer/src/ResizetizeImages.cs

```diff
@@ -11,6 +11,8 @@ namespace Microsoft.Maui.Resizetizer
 {
 	public class ResizetizeImages : MauiAsyncTask, ILogger
 	{
+		internal bool AllowVectorAdaptiveIcons = false;
```

Comment:
For testing purposes as well as to potentially bring back the vector support if there is demand.